### PR TITLE
Updates to cross-mio csvs for Irm 6, Papa 7, Pioneer 13

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Included in this pull request are updates in the following cross-mio csvs:
CruiseInformation - added Irm 6, Papa 7, Pioneer 13 cruises
vocab - added Irminger gliders (none had been deployed at Irminger previously)
platform_bulk_load-AssetRecord - added platforms for Irm 6, Papa 7, Pioneer 13
sensor_bulk_load-AssetRecord - added 1 sensor for Irm 6 glider

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
Updated sensor_bulk_load-AssetRecord again with another glider NUTNRM s/n per conversation w CDobson.

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -167,3 +167,6 @@ EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,ce02shsp_00013 dep
 EK20190703,R/V Elakha,2019-07-03T15:17:00,2019-07-03T21:00:00,ce02shsp_00014 deployment
 AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
+AR35-05,R/V Neil Armstrong,2019-08-02T00:00:00,2019-08-24T23:59:00,Irminger 6
+SKQ201920S,R/V Sikuliaq,2019-09-18T00:00:00,2019-10-05T23:59:00,Papa 7
+AR-38,R/V Neil Armstrong,2019-10-05T00:00:00,2019-10-15T23:59:00,Pioneer 13
```
Comment (snwhite) [inline cruise/CruiseInformation.csv]: Irminger 6 is scheduled to return to port on 25 August.
Pioneer 13 is AR39 (I'm agnostic on if there should be a hyphen or not) and start date is 25 September (end date is correct).

Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv]: Whoops on start date for P13 - the 5th is start of leg 2.  Will update.  
As for cruise number for P13- the draft cruise plan has AR-38 (with the dash).

Comment (rtravis-whoi) [reply cruise/CruiseInformation.csv]: I have implemented changes noted by Sheri

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1688,3 +1688,21 @@ GP05MOAS-GL469-00-ENG000000,Global Station Papa,Mobile Assets,Open Ocean Glider
...
+GI05MOAS-GL525-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 525,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+GI05MOAS-GL525-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 525,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
+GI05MOAS-GL525-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 525,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+GI05MOAS-GL525-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 525,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+GI05MOAS-GL560,Global Irminger Sea,Mobile Assets,Open Ocean Glider 560,,,,0,1000
+GI05MOAS-GL560-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 560,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+GI05MOAS-GL560-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 560,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
+GI05MOAS-GL560-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 560,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+GI05MOAS-GL560-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 560,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
```
Comment (reedan88) [inline vocab/vocab.csv]: Looks good as far as I can tell.

**Comment by rtravis-whoi** [comment]
Sheri & Andrew - I added another NUTNRM sensor s/n to sensor_bulk_load in this pull request that was identified during metadata-glider review.  This way it will be in the system when the metadata-glider PR is approved/merged.  NUTNRM s/n 730.

File: cruise/CruiseInformation.csv [hunk 3]
```diff
@@ -167,3 +167,6 @@ EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,ce02shsp_00013 dep
 EK20190703,R/V Elakha,2019-07-03T15:17:00,2019-07-03T21:00:00,ce02shsp_00014 deployment
 AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
+AR35-05,R/V Neil Armstrong,2019-08-02T00:00:00,2019-08-25T23:59:00,Irminger 6
+SKQ201920S,R/V Sikuliaq,2019-09-18T00:00:00,2019-10-05T23:59:00,Papa 7
+AR-40,R/V Neil Armstrong,2019-09-25T00:00:00,2019-10-15T23:59:00,Pioneer 13
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv]: reviewed Sheri's Pioneer 13 cruise update - looks good.

**Pull request merged**